package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "ylitse",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "16.9.0",
+    "react-native": "0.61.3"
+  },
+  "devDependencies": {
+    "@babel/core": "7.6.2",
+    "@babel/runtime": "7.6.2",
+    "@react-native-community/eslint-config": "0.0.5",
+    "@types/jest": "24.0.18",
+    "@types/react-native": "0.60.21",
+    "@types/react-test-renderer": "16.9.0",
+    "@typescript-eslint/eslint-plugin": "2.6.0",
+    "@typescript-eslint/parser": "2.6.0",
+    "babel-jest": "24.9.0",
+    "babel-plugin-module-resolver": "3.2.0",
+    "eslint": "6.5.0",
+    "eslint-config-airbnb-typescript": "6.0.0",
+    "eslint-config-prettier": "6.5.0",
+    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-react": "7.16.0",
+    "eslint-plugin-react-hooks": "2.2.0",
+    "jest": "24.9.0",
+    "metro-react-native-babel-preset": "0.56.0",
+    "prettier": "1.18.2",
+    "react-test-renderer": "16.9.0",
```

Comment:
"react-native-svg-asset-plugin": "0.3",
I can do another PR for this one : P

Reply:
Please : ) Or maybe only after we really need those